### PR TITLE
Solved tasks added

### PR DESCRIPTION
**Pull request by Alex27Khalupka**
Task1.go - finding median number for array of integers.
Task2.go - task with square.

### CODE REVIEW

File: FirstHomework/Task1.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+func median(i []int) float64{
+	sort.Ints(i)
+	var med float64
+
+	if len(i) % 2 == 1{
+		med = float64(i[len(i) / 2])
+		return med
+
+	} else {
```
Comment (ikamornikau) [inline FirstHomework/Task1.go]: Do we need `else` here? If the condition above isn't met, the flow will pass by the section.
I would prefer:
```
if len(i) % 2 == 1 {
        return float64(i[len(i) / 2])
}

return ...
```

File: FirstHomework/Task2.go [hunk 2]
```diff
@@ -0,0 +1,34 @@
+package main
+
+import "fmt"
+
+type Point struct {
```
Comment (ikamornikau) [inline FirstHomework/Task2.go:6]: Please provide a comment to describe the type.

File: FirstHomework/Task2.go [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+	x, y int
+}
+
+type Square struct {
```
Comment (ikamornikau) [inline FirstHomework/Task2.go:13]: Please add a comment to describe the type.

File: FirstHomework/Task2.go [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+	start Point
+	a uint
```
Comment (ikamornikau) [inline FirstHomework/Task2.go]: What is `a`? Could you please add a comment to provide more clarity?

File: FirstHomework/Task2.go [hunk 5]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+func (s Square) End () Point {
```
Comment (ikamornikau) [inline FirstHomework/Task2.go]: What is `End()`? What does it do? Please add more clarity by adding a comment.

File: FirstHomework/Task1.go [hunk 6]
```diff
@@ -0,0 +1,27 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func main() {
+	array := []int{9, 7, 2, 5, 3, 6}
```
Comment (ikamornikau) [inline FirstHomework/Task1.go]: Please add one more case for odd amount of elements.

**Review by ikamornikau — CHANGES_REQUESTED** [review]
Please address the requested changes.

**Comment by ikamornikau** [comment]
Please `go fmt` your code.

**Review by ikamornikau — APPROVED** [review]

File: FirstHomework/Task1.go [hunk 7]
```diff
@@ -0,0 +1,24 @@
+package main
+
+import (
+	"fmt"
+	"sort"
+)
+
+func main() {
+	EvenArray := []int{9, 7, 2, 5, 3, 6}
```
Comment (wshaman) [inline FirstHomework/Task1.go:9]: Is this array or slice?

Comment (Alex27Khalupka) [reply FirstHomework/Task1.go:9]: Slice

**Review by wshaman — APPROVED** [review]

**Pull request merged**